src/component/TodoItem.js

```diff
@@ -4,13 +4,19 @@ class ToDoItem extends Component {
     render() {
         const listItem = this.renderDOM();
```

Comment:
your organization for passing props is really clean! keep up the good work, modularity can get messy at times.